src/Http/Controllers/Api/v2/RoleController.php

```diff
@@ -209,6 +209,11 @@ public function postNew(NewRole $request)
 
         $role->save();
 
+        if ($request->has('permissions'))
```

Comment:
This need an API documentation update. Can you change the `RequestBody` definition ?
https://github.com/eveseat/api/blob/9d75c91ac73813d50efce74e08ce80654e6b9ce6/src/Http/Controllers/Api/v2/RoleController.php#L134